Gemfile

```diff
@@ -6,20 +6,13 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 # development dependencies will be added by default to the :development group.
 gemspec
 
+gem 'pagy', '~> 3.8'
```

Comment:
This needs to be added to the `gemspec` just so it will be fetched by whoever installs the gem